extensions/bombman/standalone.mjs

```diff
@@ -80,6 +89,46 @@ class Standalone {
 
     // set cooldown
     this.cooldown = new Set(); // TODO: Cooldown Manager
+
+    this.playerIDs = new Set(); // store playerID
+
+    const initTime = new Date();
+    const startTime = new Date(Math.floor((+initTime + START_GAME_INTERVAL - 1) / START_GAME_INTERVAL) * START_GAME_INTERVAL);
+    alarm(startTime, this.setGameInterval.bind(this));
+  }
+
+  /**
+   * @param {Object} player The player object
+   */
+  joinGame(player) {
+    this.playerIDs.add(player.playerID);
+  }
+
+  /**
+   * Game internal
+   */
+  setGameInterval() {
+    alarm.recurring(START_GAME_INTERVAL, this.startGame.bind(this));
+  }
+
+  /**
+   * start Game
+   * @return {Boolean} success or not
+   */
+  async startGame() {
+    // TODO:start Game
+    if (this.gameStarted) {
+      await this.resetGame();
+    }
+    if (this.playerIDs.size < 2) {
+      console.log('[bombman] player not enough to start (< 3)');
```

Comment:
Consider emitting a notification to all users participating. Search for `showNotification` and `showAnnouncement` for examples.
```
this.helper.callS2cAPI(pid, 'notification', 'showNotification', 5000, `...`);
```